lib/node_modules/@stdlib/blas/base/zcopy/README.md

```diff
@@ -0,0 +1,228 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# zcopy
+
+> Copy values from one complex double-precision floating-point vector to another complex double-precision floating-point vector.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var zcopy = require( '@stdlib/blas/base/zcopy' );
+```
+
+#### zcopy( N, x, strideX, y, strideY )
+
+Copies values from `x` into `y`.
+
+```javascript
+var Complex128Array = require( '@stdlib/array/complex128' );
+var realf = require( '@stdlib/complex/realf' );
```

Comment:
The use of `realf`and `imagf` here and elsewhere in this PR is incorrect. You should be using `@stdlib/complex/real` which is for extracting the real component of a **double-precision floating-point number**.

Reply:
The `f` suffix tells you that the API is for single-precision floating-point numbers, not double-precision.